### PR TITLE
Added introduction + reference

### PR DESCRIPTION
**Pull request by psy2848048**
For initial checking

### CODE REVIEW

File: content/docs/introduction/trading_fees.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+bookFlatSection: true
+title: Trading fees
+weight: 11
+---
+
+# Trading Fees
+
+Each liquidity pool for allowlisted asset pair has **LP Commission** rates paid as a trading fee outside of the spread algorithmic price-making determines. In Starfleit, LP Commission is fixed at *0.3%* and are deducted from the trader's received asset of the transaction.
+
+{{< katex display >}}\text{received} = B_{\text{out}}(1- \text{fee}_{\text{LP}}) - \text{tax}{{< /katex >}}
```
Comment (nowooj) [inline content/docs/introduction/trading_fees.md]: tax?

Comment (psy2848048) [reply content/docs/introduction/trading_fees.md]: https://github.com/starfleit/docs-starfleit-io/pull/1/commits/93a18df455b10001483ad5d955438478c29b52a6

File: content/_index.md [hunk 2]
```diff
@@ -6,5 +6,27 @@ title: About
...
+
+### Trader
+A **trader** can exchange their tokens for other tokens through Starfleit using the price determined by the liquidity pool ratio.
+
+### Liquidity Provider
+A **liquidity provider** adds equal value of two assets to their corresponding Starfleit pair pool, increasing the liquidity for the corresponding pair market while maintaining the pool price. In this process, liquidity providers are rewarded by newly minted LP tokens. LP tokens represent the liquidity provider's share in the pool and provide the benefit of earning trading fees, which are accumulated into the pool on every swap transaction. Liquidity providers can burn their LP tokens to withdraw their share from the pool. 
+***
+# Starfleit Web Application
+
+**Starfleit Web Application** is the official web front-end interface for swapping Terra native tokens and CW20 tokens. The web application is located [here](https://app.starfleit.io).
```
Comment (jhlee-young) [inline content/_index.md]: `Terra` native tokens

Comment (psy2848048) [reply content/_index.md]: https://github.com/starfleit/docs-starfleit-io/pull/1/commits/26c7a6eaab680daa3b5f8368836011cf45e126e3

**Review by jhlee-young — APPROVED** [review]

**Review by jbamlee — APPROVED** [review]

**Pull request merged**